build/book-byen-admin.json

```diff
@@ -0,0 +1,96 @@
+[
```

Comment:
Consider [marking everything in the `build` dir as generated](https://github.com/github/linguist/blob/master/docs/overrides.md).